CGPs/cgp-0021.md

```diff
@@ -1,6 +1,6 @@
 ---
 cgp: 21
-title: Update `randomnessBlockRetentionWindow` to Extend Attestation Expiration Duration
+title: Update randomnessBlockRetentionWindow to Extend Attestation Expiration Duration
```

Comment:
The titles ideally free of special characters like backticks to avoid issues when rendering these strings